packages/ui-progress/src/ProgressBar/index.tsx

```diff
@@ -130,11 +131,11 @@ class ProgressBar extends Component<ProgressBarProps> {
           />
 
           <span css={styles?.track} role="presentation" aria-hidden="true">
-            <span css={styles?.trackValue}></span>
+            <span css={styles?.trackValue}>{renderValueInside && value}</span>
           </span>
         </span>
 
-        {value && (
+        {value && !renderValueInside && (
```

Comment:
Can you try to write a test for this? e.g if renderValueInside is `true` then the value text should be inside the track's DOM

Reply:
done